templates/artist/includes/invest.html

```diff
@@ -0,0 +1,35 @@
+{% load staticfiles %}
+{% load humanize %}
+
+{% if latest_campaign %}
+    {% with campaign=latest_campaign %}
+        {% if campaign.open %}
+            {% if request.user.is_authenticated %}
+                <img src="{% static "img/project.png" %}" alt="project" height="200px" width="200px" />
+                <h5 class="remaining">{{ campaign.num_shares_remaining }} shares remaining </h5>
+                <button id="invest-button" class="button buy" type="button">Invest ${{ DEFAULT_MIN_PURCHASE|floatformat:2 }}</button>
+                <div class="invest-num-shares">
+                    <button id="remove-shares" class="secondary hollow button disabled shares" type="button" disabled>-</button>
+                    <input type="number" min="1" max="{{ campaign.num_shares_remaining }}" value="{{ campaign.default_num_shares }}" />
+                    <button id="add-shares" class="secondary hollow button" type="button">+</button>
+                </div>
+                <h5 class="total-amount">Total: <span data-tooltip aria-haspopup="true" class="has-tip [tip-right] [radius round]" title="${{ PERDIEM_FEE }} PerDiem + Credit Card Fee (2.9%+$0.30)"><span id="shares-price"></span></span></h5>
+                <div class="break">
+                    <hr>
+                </div>
+                <h6>{{ artist.name }} is raising ${{ campaign.amount|intcomma }} and giving {{ campaign.fans_percentage }}% of sales from {{ campaign.project.reason }} to investors</h6>
+                <h6>The value of a share is ${{ campaign.value_per_share }}</h6>
+                <h6>Each share is worth {{ campaign.share_price }} of {{ campaign.project.reason }} revenue</h6>
+                <h6>This project is valued at $<span>{{ campaign.valuation }}</span></h6>
+            {% else %}
+                {% include "artist/includes/auth_buttons.html" %}
+                <h4>To invest in {{ artist.name }}</h4>
+            {% endif %}
+            {% else %}
```

Comment:
This one line needs to be unindented once. It's because the statement `{% else %}` is part of the logical control flow for `{% if campaign.open %}`.
```html
{% if campaign.open %}
    {% if request.user.is_authenticated %}
        <!-- Show invest buttons -->
    {% else %}
        <!-- Convince user to sign in -->
    {% endif %}
{% else %}
    {% if campaign.percentage_funded == 100 %}
        <!-- Campaign was sold out -->
    {% endif %}
    <!-- Either way, the campaign will no longer be available on such and such date -->
{% endif %}
```
In an even more abstract sense:
```
if the campaign is open:
    allow the user to invest, or convince them to be a user
else:
    tell the user when the campaign ended, and if it was funded
```

Do you see how the else statement explains what happens when the `if` is false? The indenting is key to communicating that effectively.